### DISCUSSION TITLE
Support for Multisite?

### DISCUSSION THREAD

**Discussion by doodirock** [wpengine/faustjs#1521] [General]
Are there any examples (or is it possible) to use Faust with a multisite?  Our goal would be to serve many different sites with very similar backend requirements on several different unique frontend Next.js apps on different domains.  This is more ideal instead of creating multiple WP installs just to have almost identical fields, API structure etc.  

Is this possible currently or would there be issues with things like previews, etc.

**Comment by josephfusco** [comment]
Hey @doodirock, given your use case, Faust can absolutely be used within a WordPress multisite!

If the [Faust.js WordPress plugin](https://wordpress.org/plugins/faustwp/) (and [WPGraphQL](https://wordpress.org/plugins/wp-graphql/)) are network activated within your multisite, each individual 'blog' within the network can be coupled to a single front-end application. It's essentially 'business as usual' for configuring your site, as Faust does not currently have any multisite-specific features.

Hope that helps!

**Reply by doodirock** [reply]
Thats amazing! How would this work with different preview URLS for each of those sites?  I assume each multisite would need its own preview setting to be able to manage previews.

**Reply by josephfusco** [reply]
Correct, each site within the multisite network would assign it's own front-end URL within the plugin settings which are scoped to that site, as each site would have it's own GraphQL endpoint.

The WordPress site managing the content for [faustjs.org](https://faustjs.org/) is actually a single site within a multisite network! We are using a subdirectory multisite, but Faust.js should also work as expected with a subdomain setup.

**Reply by doodirock** [reply]
Great news Thanks so much for the work your team is doing to modernize WP.  Cheers!

**Comment by asimplify** [comment]
Can anyone provide more information for this? I am getting 404 when trying to use multisite
